components/desktop/dmenu/patches/01-config-mk.patch

```diff
@@ -0,0 +1,17 @@
+--- dmenu-4.9/config.mk.orig	2019-02-02 13:55:02.000000000 +0000
++++ dmenu-4.9/config.mk	2020-08-09 10:54:50.168251925 +0000
+@@ -2,7 +2,7 @@
+ VERSION = 4.9
+ 
+ # paths
+-PREFIX = /usr/local
++PREFIX = /usr
+ MANPREFIX = $(PREFIX)/share/man
+ 
+ X11INC = /usr/X11R6/include
+@@ -28,4 +28,4 @@
+ LDFLAGS  = $(LIBS)
+ 
+ # compiler and linker
+-CC = cc
++CC = gcc
```

Comment:
Should we just remove `CC = cc` or set to `CC ?= gcc` to use oi-userland gcc, not any?